WikibaseDataModel.php

```diff
@@ -7,7 +7,10 @@
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
 
-define( 'WIKIBASE_DATAMODEL_VERSION', '5.0.0 alpha' );
+/**
+ * @deprecated since 5.0
```

Comment:
@thiemo the constant tells you the version. 

In https://github.com/ProjetPP/WikibaseEntityStore/blob/master/src/Cache/EntityIdForTermCache.php#L81 it is used as part of a cache key, which seems a valid use case to support